### ISSUE TITLE
dead_body item is now a hanging skeleton

### ISSUE THREAD

**Issue by JayH2971** [Interkarma/daggerfall-unity#2467]
Version: 0.13.5. No mods, no texture replacements. Extracted a fresh DaggerfallGameFiles to be double-sure.

Previously, the quest item "dead_body" was a bloody corpse on the ground (see [Standard Protection](https://en.uesp.net/wiki/File:DF-quest-Standard_Protection.jpg).) I don't know when that changed, but now when the quest system uses dead_body, it creates a hanging skeleton instead: see https://imgur.com/a/Px55jsl.

**Comment by Interkarma** [comment]
Spooky! The last change I can see to this item data is [in 2017](https://github.com/Interkarma/daggerfall-unity/commit/f070e754d0a65f747e71b1173691abc5be869dd4). The _dead_body_ item template still has the correct archive/record of 380/1 in master code at this time.

No idea why it presented as something else in-game. I'll need to trace through the steps to see what's happening.

**Comment by JayH2971** [comment]
A user on Discord also told me they have a skeleton, so I'm certain this isn't just my machine being odd.

**Comment by JorisVanEijden** [comment]
I think you're referring to quest R0C11Y28.
The skeleton is not the "Item _I.04_ dead_body" but the "Item _I.08_ item class 26 subclass 3"

When I look at the original data in my querst viewer (https://stellargames.github.io/Quester/#R0C11Y28) I think I know what's going on.

The skeleton is supposed to be the autopsied remains of an earlier victim.
The dead body is supposed to arrive at the same location at a later time, representing a new victim.

I think in the original the quest items and npc's were not supposed to spawn at the exact same location, which is why it looks a bit weird in DFU.

**Comment by JayH2971** [comment]
> I think you're referring to quest R0C11Y28.
> The skeleton is not the "Item I.04 dead_body" but the "Item I.08 item class 26 subclass 3"

That may be the case, but even in custom quests, attempting to use the dead_body corpse creates the skeleton diagram in its place. Some index definitely got shifted over in the last few months. I used dead_body for the first couple of years without problem, and this sprite change is fairly recent.

**Comment by Interkarma** [comment]
I haven't been able to reproduce this in current master. Dead body shows up normally for me every time. I tested with a couple quests using the dead_body item definition (Standard Protection and Elysana's Robe when Castellian is slain).

**Current Master**
![image](https://user-images.githubusercontent.com/10426244/207463621-e8c434d6-5834-4c67-bbff-ca03ccb82603.png)

I also don't reproduce in live 0.14.5 release.

**0.14.5**
![image](https://user-images.githubusercontent.com/10426244/207464427-818430d9-c212-4596-9cbe-f6528b29e2c7.png)

Not sure how this is coming about for you, I'm sorry. :( Are you sure there's no other mods or changes in the mix? Could I please ask for you to download a fresh copy of DaggerfallGameFiles and DFU, unzip both to fresh new folders, and try again?

**Comment by JayH2971** [comment]
Well, this is a shame. Sorry for wasting your time. I just tested it in 0.14.5 with no mods fresh install, and it's a dead body... I have no idea how I got a skeleton graphic mixed up in my stuff, but maybe I'll take some time looking through my files and seeing if something did that.
![Screenshot at 2022-12-13 16-16-55](https://user-images.githubusercontent.com/37541932/207465963-a80df650-c8fd-4795-a832-9f5618c2ea5b.png)
